### DISCUSSION TITLE
Background color is wrong

### DISCUSSION THREAD

**Discussion by hunt-g** [arcticicestudio/nord#246] [Help & Support] [answered]
The primary/background color on vs code doesn't match other Nord themes or the official colors. The background is #2e343f instead of #2e3440 for some reason.  

![image](https://user-images.githubusercontent.com/61890674/131200570-6ab42ee0-307f-4e06-8ce9-276ad2760798.png)

**Comment by hunt-g** [comment]
https://github.com/microsoft/vscode/issues/114591

**Comment by svengreb** [comment, marked as answer]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

---

@hunt-g Thanks again for your contributions 🚀
This was a larger topic in the _VS Code_ community after a specific update that changed the rendering. Like you already mentioned the issue, and root cause, is documented in [microdoft/vscode#114591](https://github.com/microsoft/vscode/issues/114591). Therefore I'll close this discussion, but feel free to open a new one if the problem this exists.
